HttpClient/FirstApp/src/app/user.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {map} from 'rxjs/operators';
+import {User} from './user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  constructor(private http: HttpClient) {}
+
+  getDataUser(): any {
```

Comment:
```suggestion
  getDataUser(): Observable<User> {
```

Or other observable. HttpClient always returns an Observable object